tests/phpunit/api/v3/PaymentTest.php

```diff
@@ -620,10 +620,11 @@ public function testUpdatePayment() {
       'financial_trxn_id' => $payment['id'] - 1,
     ];
 
-    $eft = $this->callAPISuccess('EntityFinancialTrxn', 'get', $minParams);
+    $eft = $this->callAPISuccess('EntityFinancialTrxn', 'get', $minParams)['values'];
+    $this->assertCount(2, $eft);
```

Comment:
This assertCount was missing & the check below only works if there are $eft records